docs/data/system/components/container/container.md

```diff
@@ -7,7 +7,7 @@ githubLabel: 'component: Container'
 
 # Container
 
-<p class="description">The container centers your content horizontally. It's the most basic layout element.</p>
+<p class="description">The Container component centers your content horizontally. It's the most basic layout element.</p>
```

Comment:
To match 1:1 the Material UI page.

```suggestion
<p class="description">The Container component centers primary layout elements with a fluid or fixed width.</p>
```

---

Overtime, we can kill the Material UI page, I belive forcing people to MUI System would be best (forcing them to separate in their mind Material UI and MUI System), like they would between Tailwind CSS and Shadcn UI.